rosetta/rosetta.sh

```diff
@@ -59,36 +66,37 @@ echo -e
 docker-compose --env-file .env up -d
 ;;
 
-'testnet')
+'devnet')
 #Create the folder structure for the observer stack
-mkdir -p ${STACK_FOLDER_TESTNET}/{proxy,node-0,node-1,node-2,node-metachain}/{config,logs}
-mkdir -p ${STACK_FOLDER_TESTNET}/{node-0,node-1,node-2,node-metachain}/db
-mkdir -p ${KEYS_FOLDER_TESTNET}
+mkdir -p ${STACK_FOLDER_DEVNET}/{proxy,node-0,node-1,node-2,node-metachain}/{config,logs}
+mkdir -p ${STACK_FOLDER_DEVNET}/{node-0,node-1,node-2,node-metachain}/db
+mkdir -p ${KEYS_FOLDER_DEVNET}
 
 #Clone the repo and cd there
 if [ -d "$GIT_HOME" ]; then sudo rm -rf $GIT_HOME; fi
 git clone -b master https://github.com/ElrondNetwork/observing-squad.git $GIT_HOME
-cd $GIT_HOME/rosetta-testnet
+cd $GIT_HOME/rosetta-devnet
 
 if [[ -f ~/.proxyDockerTag ]]; then
+  echo \ >> .env
   sed -i '/PROXY_TAG/d' .env
-  echo PROXY_TAG=${PROXY_TAG} >> .env
+  echo PROXY_TAG="${PROXY_TAG}" >> .env
 fi
 
 #Generate Keys and place them in their respective folders
 
-for OBSERVER_TESTNET in "${NODES[@]}"
+for OBSERVER_DEVNET in "${NODES[@]}"
 do
-   echo -e "${GREEN}--> Generating key for observer ${CYAN}$OBSERVER_TESTNET${GREEN}...${NC}"
-   docker run --rm --mount type=bind,source=${KEYS_FOLDER_TESTNET},destination=/keys --workdir /keys elrondnetwork/elrond-go-keygenerator:latest && sudo chown $(whoami) ${KEYS_FOLDER_TESTNET}/validatorKey.pem && mv ${KEYS_FOLDER_TESTNET}/validatorKey.pem ${STACK_FOLDER_TESTNET}/node-$OBSERVER_TESTNET/config/observerKey_$OBSERVER_TESTNET.pem
+   echo -e "${GREEN}--> Generating key for observer ${CYAN}$OBSERVER_DEVNET${GREEN}...${NC}"
+   docker run --rm --mount type=bind,source=${KEYS_FOLDER_DEVNET},destination=/keys --workdir /keys elrondnetwork/elrond-go-keygenerator:latest && sudo chown $(whoami) ${KEYS_FOLDER_DEVNET}/validatorKey.pem && mv ${KEYS_FOLDER_DEVNET}/validatorKey.pem ${STACK_FOLDER_DEVNET}/node-$OBSERVER_DEVNET/config/observerKey_$OBSERVER_DEVNET.pem
 done
 
 #Say what has been started
-echo "testnet" > ~/.squadlocation
+echo "devnet" > ~/.squadlocation
 
 #Start the stack
 echo -e
-echo -e "${GREEN}--> Starting the Observer+Proxy TestNet Stack...${NC}"
+echo -e "${GREEN}--> Starting the Observer+Proxy DEVNET Stack...${NC}"
```

Comment:
should we keep this for testnet as well?

Reply:
no. just for devnet and mainnet because testnet is reset more ofen